CMake/find_cuda.cmake

```diff
@@ -0,0 +1,17 @@
+#configure for CUDA
+set(tmp OFF)
+include(CheckLanguage)
+check_language(CUDA)
+if (CMAKE_CUDA_COMPILER AND ((DEFINED ENABLE_CUDA AND ENABLE_CUDA)
+    OR (NOT DEFINED ENABLE_CUDA)))
+    enable_language(CUDA)
+    message(STATUS "CUDA features -- enabled")
+    set(tmp ON)
+    enable_language(CUDA)
+elseif (REQUIRE_CUDA)
+    message(FATAL_ERROR "CUDA features -- required but not found.")
+else()
+    message(STATUS "CUDA features -- not found.")
+endif()
+set(LINSOLVE_HAVE_CUDA ${tmp} CACHE BOOL "Set when we are using CUDA")
+set(LINSOLVE_CUDA_ARCHITECTURES "60;70;75" CACHE STRING "Compile for these CUDA architectures")
```

Comment:
Don't have variables with LINSOLVE in the name. That's just lifted from Parth's project.

Reply:
Fair.